logicrunner/builtin/contract/member/member.go

```diff
@@ -458,50 +465,67 @@ func (m *Member) migration(txHash string, burnAddress string, amount big.Int, un
 	return strconv.Itoa(int(confirms)), nil
 }
 
-// FindDeposit finds deposits for this member with this transaction hash.
-func (m *Member) FindDeposit(txHash string, inputAmountStr string) (bool, deposit.Deposit, error) {
+// GetDeposits get all deposits for this member
+func (m *Member) GetDeposits() ([]map[string]string, error) {
+	var result []map[string]string
+	for _, dRef := range m.Deposits {
+
+		d := deposit.GetObject(dRef)
+
+		depositInfo, err := d.MapMarshal()
```

Comment:
Its better to use smth like DepositResoinse object with tags for json marshaling in GetBalanceResponse